sdk/containerregistry/azure-containerregistry/azure/containerregistry/_container_registry_client.py

```diff
@@ -806,33 +808,28 @@ def upload_manifest(
         return digest
 
     @distributed_trace
-    def upload_blob(self, repository, data, **kwargs):
-        # type: (str, IO, **Any) -> str
+    def upload_blob(self, repository: str, data: IO, **kwargs) -> Tuple[str, int]:
         """Upload an artifact blob.
 
-        :param str repository: Name of the repository
+        :param str repository: Name of the repository.
         :param data: The blob to upload. Note: This must be a seekable stream.
         :type data: IO
-        :returns: The digest of the uploaded blob, calculated by the registry.
-        :rtype: str
+        :returns: The digest and size of the uploaded blob
+        :rtype: Tuple[str, str]
```

Comment:
Tuple[str, int]?

Reply:
Good catch, the later one should be int. Thanks!